proto/buf/alpha/registry/v1alpha1/role.proto

```diff
@@ -29,6 +29,8 @@ enum OrganizationRole {
   ORGANIZATION_ROLE_OWNER = 1;
   ORGANIZATION_ROLE_ADMIN = 2;
   ORGANIZATION_ROLE_MEMBER = 3;
+  // ORGANIZATION_ROLE_WRITER is not supported in certain environments.
```

Comment:
This comment is isn't helpful and imo will only evoke questions. We don't describe any of the impact/availability of any of the other roles and I don't think we should start with one off comments like this.

I think it is fine if people attempt to use this and receive an error message if the role isnt available in the instance.

Reply:
removed 👍